include/deal2lkit/sundials_interface.h

```diff
@@ -73,18 +98,34 @@ public :
                                      const unsigned int step_number,
                                      const double h);
 
-  /** Compute the residual dst = F(t, y, y_dot). */
+  /**
+   * Given @p t, @p y, and @p y_dot, this function evalutes the residual of
+   * @f[
+   *   F(t,y,\dot{y}) = 0
+   * @f]
+   * and stores the result in the variable @p dst.
+   */
   virtual int residual(const double t,
                        const VEC &y,
-                       const VEC &sy_dot,
+                       const VEC &sy_dot, // TODO: Is sy_dot correct?
```

Comment:
change the name according to your documentation. Indeed the name of variable given here are not needed since they can be overwritten in the .cc file